Some images are missing in 2.2.0 tarball.
This might be fixed in later versions.

--- IlmImfTest/testSampleImages.cpp	2016-11-29 09:30:15.387433213 +0000
+++ IlmImfTest/testSampleImages.cpp	2016-11-29 09:30:36.662335997 +0000
@@ -162,7 +162,7 @@
 	compareImages (ILM_IMF_TEST_IMAGEDIR "comp_b44.exr",
 		       ILM_IMF_TEST_IMAGEDIR "comp_b44_piz.exr");
 
-	compareImages (ILM_IMF_TEST_IMAGEDIR "comp_dwaa_v1.exr",
+	/* compareImages (ILM_IMF_TEST_IMAGEDIR "comp_dwaa_v1.exr",
 		       ILM_IMF_TEST_IMAGEDIR "comp_dwaa_piz.exr");
 	compareImages (ILM_IMF_TEST_IMAGEDIR "comp_dwaa_v2.exr",
 		       ILM_IMF_TEST_IMAGEDIR "comp_dwaa_piz.exr");
@@ -170,7 +170,7 @@
 	compareImages (ILM_IMF_TEST_IMAGEDIR "comp_dwab_v1.exr",
 		       ILM_IMF_TEST_IMAGEDIR "comp_dwab_piz.exr");
 	compareImages (ILM_IMF_TEST_IMAGEDIR "comp_dwab_v2.exr",
-		       ILM_IMF_TEST_IMAGEDIR "comp_dwab_piz.exr");
+		       ILM_IMF_TEST_IMAGEDIR "comp_dwab_piz.exr"); */
 
 
 	cout << "ok\n" << endl;
